### ISSUE TITLE
Overlapping boxes

### ISSUE THREAD

**Issue by xiejia1995** [exc-asia-and-europe/ecpo-annotator#103]
![2022-04-29 14_49_22-Window](https://user-images.githubusercontent.com/76219575/165947794-017e0f57-3ad8-404e-9657-4176cc6de564.png)
as this pic shows, the two small rectangles ("一〇八" and "李涵秋著禁止转载") are overlapped by the polygonal shape. In this case, we can not select the two unless we move the polygon...

**Comment by MatthiasArnold** [comment]
yes. the z-index used to be an issue from the beginning. i do not think it was ever fixed. but since the annotator seems to be finally usable again, including the grouping, we can leave that as a feature request for the future.

**Comment by MatthiasArnold** [comment]
this issue has been documented earlier: 
https://github.com/exc-asia-and-europe/ecpo-annotator/issues/31
and 
https://github.com/exc-asia-and-europe/ecpo-annotator/issues/85
